pipeline/emailing.py

```diff
@@ -10,18 +10,25 @@
 from load import create_connection
 
 
-def send_email(plant_id: int, value: float, value_type: str) -> None:
-    """Sends an email to the correct botanist if the plants conditions are worrying."""
+def send_email(plant_id: int, value: float, value_type: str, condition:str ) -> None:
+    """
+    Sends an email to the correct botanist if the plants conditions are worrying.
+    Example inputs - plant_id: 1, value: 10000, value_type: 'temperature', condition: 'exceeded'.
+    """
+
     load_dotenv()
-    html = generate_html(plant_id, value, value_type)
+    html = generate_html(plant_id, value, value_type, condition)
     client = boto3.client("ses", region_name="eu-west-2")
     message = MIMEMultipart()
-    message["Subject"] = f"Plant {plant_id} Alert!"
+    
+    message["Subject"] = f"PLANT ALERT! ACCEPTABLE {value_type.upper()} LEVEL {condition.upper()}"
     body = MIMEText(
         html,
         "html")
     message.attach(body)
 
+    # The addresses the emails will be sent, 
+    # should be specified in your local env variables.
```

Comment:
Updated where the emails will be sent, not to my test email, but to the emails stored locally in an .env file.